### DISCUSSION TITLE
Adding a new function to agr.R

### DISCUSSION THREAD

**Discussion by SunSummoner** [luukvdmeer/sfnetworks#251] [Ideas]
Hey Everyone! 

I wanted to contribute to this project and I added a new function is_null_active() to remove the if loop. Would it be fine if a create pull request to review the changes?

**Comment by agila5** [comment]
> Would it be fine if a create pull request to review the changes?

Please do! Check also the [contributing guidelines](https://github.com/luukvdmeer/sfnetworks/blob/main/CONTRIBUTING.md).

**Reply by SunSummoner** [reply]
Okay! Thank you!
